### PR TITLE
Mobilenet-v3 related changes

### PR DESCRIPTION
**Pull request by slahiruk**
Fixing this --> https://github.com/ctuning/ck-mlperf/issues/47 for Mobilenet V3

### CODE REVIEW

File: package/model-tf-and-tflite-mlperf-mobilenet-v3/.cm/meta.json [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+      "*MODEL_IMAGE_HEIGHT": "$<<MODEL_MOBILENET_RESOLUTION>>$",
+      "*MODEL_IMAGE_WIDTH": "$<<MODEL_MOBILENET_RESOLUTION>>$",
+      "*PACKAGE_NAME": "v$<<MODEL_MOBILENET_VERSION>>$-$<<MODEL_MOBILENET_TYPE>>$_$<<MODEL_MOBILENET_RESOLUTION>>$_$<<MODEL_MOBILENET_MULTIPLIER>>$_$<<MODEL_MOBILENET_PRECISION>>$.tgz",
+      "*PACKAGE_NAME_MOBILENET_V3":"v$<<MODEL_MOBILENET_VERSION>>$-$<<MODEL_MOBILENET_TYPE>>$_$<<MODEL_MOBILENET_RESOLUTION>>$_$<<MODEL_MOBILENET_MULTIPLIER>>$_$<<MODEL_MOBILENET_PRECISION>>$",
+      "*WEIGHTS_FILE": "ema/model.ckpt",
+      "ML_MODEL_DATA_LAYOUT": "NHWC",
+      "MODEL_INPUT_LAYER_NAME": "input",
+      "MODEL_MOBILENET_VERSION": "3",
+      "MODEL_NORMALIZE_DATA": "YES",
+      "MODEL_OUTPUT_LAYER_NAME": "MobilenetV3/Predictions/Reshape_1",
```
Comment (psyhtest) [inline package/model-tf-and-tflite-mlperf-mobilenet-v3/.cm/meta.json]: "MobilenetV3/Predictions/Softmax"

File: package/model-tf-and-tflite-mlperf-mobilenet-v3/.cm/meta.json [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+      "extra_env": {
+        "MODEL_MOBILENET_TYPE":"large",
+        "MODEL_MOBILENET_PRECISION":"float",
+        "MODEL_MOBILENET_MULTIPLIER": "1.0",
+        "MODEL_MOBILENET_RESOLUTION": "224"
+      },
+      "extra_tags": [
+        "multiplier.1.0",
+        "side.224",
+	"large",
```
Comment (psyhtest) [inline package/model-tf-and-tflite-mlperf-mobilenet-v3/.cm/meta.json]: indent >>

File: package/model-tf-and-tflite-mlperf-mobilenet-v3/.cm/meta.json [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+  "variations": {
+    "from.googleapis": {
+      "extra_env": {
+        "PACKAGE_URL": "https://storage.googleapis.com/mobilenet_v3/checkpoints"
+      },
+      "on_by_default":"yes"
+    },
+    "v3-large_224_1.0_float": {
+      "extra_env": {
+        "MODEL_MOBILENET_TYPE":"large",
```
Comment (psyhtest) [inline package/model-tf-and-tflite-mlperf-mobilenet-v3/.cm/meta.json]: space after `:` please

Comment (psyhtest) [reply package/model-tf-and-tflite-mlperf-mobilenet-v3/.cm/meta.json:60]: `MODEL_MOBILENET_TYPE` -> `MODEL_MOBILENET_SIZE`

File: package/model-tf-and-tflite-mlperf-mobilenet-v3/.cm/meta.json (hunk 2 above)
Comment (psyhtest) [reply package/model-tf-and-tflite-mlperf-mobilenet-v3/.cm/meta.json:68]: `large` -> `size.large`

Comment (psyhtest) [reply package/model-tf-and-tflite-mlperf-mobilenet-v3/.cm/meta.json]: `large` -> `kind.large`

File: package/model-tf-and-tflite-mlperf-mobilenet-v3/.cm/meta.json (hunk 3 above)
Comment (psyhtest) [reply package/model-tf-and-tflite-mlperf-mobilenet-v3/.cm/meta.json]: `MODEL_MOBILENET_TYPE` -> `MODEL_MOBILENET_KIND`

File: package/model-tf-and-tflite-mlperf-mobilenet-v3/.cm/meta.json (hunk 1 above)
Comment (slahiruk) [reply package/model-tf-and-tflite-mlperf-mobilenet-v3/.cm/meta.json]: Resolved with commit https://github.com/ctuning/ck-mlperf/pull/52/commits/00dec7531605b9e2852f17e611ca6eb32b023ade

File: package/model-tf-and-tflite-mlperf-mobilenet-v3/.cm/meta.json (hunk 2 above)
Comment (slahiruk) [reply package/model-tf-and-tflite-mlperf-mobilenet-v3/.cm/meta.json]: Indent resolved with --> https://github.com/ctuning/ck-mlperf/pull/52/commits/a41ac5dbb76af90bc5be81020829692a54a97fd6

`large` --> `kind.large` resolved with -->  https://github.com/ctuning/ck-mlperf/pull/52/commits/53dc263a7ad1066d1ab6ef3a1545a3c63e990bc2

File: package/model-tf-and-tflite-mlperf-mobilenet-v3/.cm/meta.json (hunk 3 above)
Comment (slahiruk) [reply package/model-tf-and-tflite-mlperf-mobilenet-v3/.cm/meta.json]: space after  `:` resolved with --> https://github.com/ctuning/ck-mlperf/pull/52/commits/abce157ab065a3a4a7ad266bbe40ce92620b9f4e

`MODEL_MOBILENET_TYPE` -->  `MODEL_MOBILENET_KIND` resolved with -->https://github.com/ctuning/ck-mlperf/pull/52/commits/c9c94616731d968b63b8e8e6b12c58d6f6207a06

**Comment by psyhtest** [comment]
@slahiruk Many thanks for your contribution!

**Pull request merged**